docs/index.md

```diff
@@ -32,7 +32,9 @@ load("@rules_rust//rust:repositories.bzl", "rust_repositories")
 rust_repositories()
 ```
 
-The rules are under active development, as such the lastest commit on the master branch should be used. `master` currently requires Bazel >= 3.0.0.
+The rules are under active development, as such the lastest commit on the `main` branch should be used. `main` is only tested
+against `4.0.0` as the minmum supported version of Bazel. Though previous versions may still be supported in certain
```

Comment:
s/minmum/minimum

I'd also wrap lines at 80, but anything under 120 (which I guess is the width of the GitHub diff view) works for me.

Reply:
wrapped to 80 (for the affected lines)